kibbeh/src/ui/mobile/AccountOverlay.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useEffect, useRef, ReactNode } from "react";
+import Draggable, {
+  DraggableBounds,
+  ControlPosition,
+  DraggableEventHandler,
+} from "react-draggable";
+
+export interface AccountOverlyProps {
+  children: ReactNode;
+  position: ControlPosition;
+  dragHandler: DraggableEventHandler;
+  closeHandler: () => void;
+  bounds: DraggableBounds;
+  className?: string;
+  axis?: "both" | "x" | "y" | "none" | undefined;
+}
+
+export const AccountOverlay: React.FC<AccountOverlyProps> = ({
+  children,
+  position,
+  bounds,
+  dragHandler,
+  closeHandler,
+  className = "",
+  axis = "both",
```

Comment:
Just a minor fix, this wouldn't be used for any xy dragging.
```suggestion
  axis = "y",
```